site/docs/hosting/deno-deploy.md

```diff
@@ -48,7 +48,7 @@ Here, we are using the bot token (`/<bot token>`).
 ### Method 1: With GitHub
 
 > This is the recommended method, and the easiest one to go with.
-> The main advantage of following this method is that Deno Deploy will watch for changes in your repository which includes your bot code, and it will deploy new versions automatically.
+> The main advantage of following method is that Deno Deploy will watch for changes in your repository which includes your bot code, and it will deploy new versions automatically.
```

Comment:
The sentence was correct before, _following this method_ is what has the discussed advantage.

The sentence is incorrect now, if anything, it would have to be _of the following method_—but that's a very different statement then, and it's not what's meant here.

Please revert this one.